### PR TITLE
feat: [C4R-817] public send mail function

### PR DESCRIPTION
**Pull request by OlteanuCosmin**
Supports only mails sent to contact mail for now, will be changed later if needed.

### CODE REVIEW

File: backend/src/mail/services/mail.service.ts [hunk 1]
```diff
@@ -13,8 +14,14 @@ export class MailService {
   ) {}
 
   async sendEmail(email: ISendMailOptions) {
+    let from: string | Address;
```
Comment (birloiflorian) [inline backend/src/mail/services/mail.service.ts:17]: Aici am putea folosi ternary operator

File: backend/src/modules/_publicAPI/public-api.controller.ts [hunk 2]
```diff
@@ -1,19 +1,23 @@
+import { ISendMailOptions } from '@nestjs-modules/mailer';
```
Comment (birloiflorian) [inline backend/src/modules/_publicAPI/public-api.controller.ts:1]: Unused import?

**Review by birloiflorian — CHANGES_REQUESTED** [review]

File: backend/src/modules/_publicAPI/public-api.controller.ts (hunk 2 above)
Comment (OlteanuCosmin) [reply backend/src/modules/_publicAPI/public-api.controller.ts]: Removed unused imports.

File: backend/src/mail/services/mail.service.ts (hunk 1 above)
Comment (OlteanuCosmin) [reply backend/src/mail/services/mail.service.ts]: Replaced with ternary operator.

File: backend/src/modules/_publicAPI/public-api-send-mail.service.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@nestjs/common';
+import { MailService } from 'src/mail/services/mail.service';
+import { ContactMailDto } from './dto/contact-mail.dto';
+
+@Injectable()
+export class PublicApiSendMailService {
+  constructor(private readonly mailService: MailService) {}
+
+  public async sendMail(mailOptions: ContactMailDto): Promise<void> {
```
Comment (radulescuandrew) [inline backend/src/modules/_publicAPI/public-api-send-mail.service.ts:9]: Nu cred ca e necesar service asta. Poti sa injectezi mailService direct in controller momentan (pana facem event based)

Comment (OlteanuCosmin) [reply backend/src/modules/_publicAPI/public-api-send-mail.service.ts]: Sters service-ul.

**Review by birloiflorian — APPROVED** [review]

File: backend/src/modules/_publicAPI/public-api-send-mail.service.ts (hunk 3 above)
Comment (OlteanuCosmin) [reply backend/src/modules/_publicAPI/public-api-send-mail.service.ts]: Sters import-ul din modul.

**Pull request merged**